### PR TITLE
mention services view in yast sw_single

### PR DESCRIPTION
**Pull request by taroth21**
### Description
Implement Fate#320573: [POST-GM] [R] Add 'services' view to yast2 sw_single

### Checklist
* Check all items that apply.

*Are backports required?*

- [ ] To maintenance/SLE12SP3
- [ ] To maintenance/SLE12SP4
- [x] To maintenance/SLE15SP0

### CODE REVIEW

File: xml/yast2_sw.xml [hunk 1]
```diff
@@ -275,6 +275,17 @@
       </para>
      </listitem>
     </varlistentry>
+    <varlistentry>
+     <term>Services</term>
+     <listitem>
+      <para>
+       Shows which packages belong to a certain service or module. Select the
```
Comment (cwickert) [inline xml/yast2_sw.xml]: service or module -> module or extension

File: xml/art_sle_modules_quick.xml [hunk 2]
```diff
@@ -1152,8 +1152,16 @@ Package  Module or Repository
 emacs    Basesystem Module (sle-module-basesystem/15/x86_64)
 emacs    Available</screen>
   </example>
- </sect1>
 
+  <para>
+   If you do not want to search for a certain package, but want to view the list
```
Comment (cwickert) [inline xml/art_sle_modules_quick.xml]: How about: "Instead of searching for packages, you can also view the list..."?

File: xml/yast2_sw.xml [hunk 3]
```diff
@@ -275,6 +275,17 @@
       </para>
      </listitem>
     </varlistentry>
+    <varlistentry>
+     <term>Services</term>
```
Comment (cwickert) [inline xml/yast2_sw.xml]: Whoever came up with the term "services" for modules and extensions should be punished. :-/

File: xml/yast2_sw.xml [hunk 4]
```diff
@@ -275,6 +275,17 @@
...
+     <listitem>
+      <para>
+       Shows which packages belong to a certain service or module. Select the 
+       module or extension (for example, <literal>Basesystem</literal> or
```
Comment (cwickert) [inline xml/yast2_sw.xml]: If you apply the previous change, "module or extension" would be repeated. :-(
Change to "Select one to display"?

File: xml/art_sle_modules_quick.xml (hunk 2 above)
Comment (taroth21) [reply xml/art_sle_modules_quick.xml]: Much better, thanks.

File: xml/yast2_sw.xml (hunk 3 above)
Comment (taroth21) [reply xml/yast2_sw.xml]: Yes, adding yet another term does not really help here. ;)

File: xml/yast2_sw.xml (hunk 1 above)
Comment (taroth21) [reply xml/yast2_sw.xml]: Good catch, thanks.

File: xml/yast2_sw.xml (hunk 4 above)
Comment (taroth21) [reply xml/yast2_sw.xml]: I decided for 'Select one entry to display...'

**Pull request merged**